Modules/HTTP/Sources/HTTP/Parser/MessageParser.swift

```diff
@@ -0,0 +1,316 @@
+import CHTTPParser
+import Core
+
+public class MessageParser: Parser {
```

Comment:
Style nit. Let's always add a whitespace before the colon on type conformances to follow the standard library style.

```swift
public class MessageParse : Parser { ... }
                          ^
```